### ISSUE TITLE
openstack4j version 3.2.0 getting com.fasterxml.jackson.databind.JsonMappingException: Can not deserialize instance of java.lang.String out of START_OBJECT token

### ISSUE THREAD

**Issue by gilbirenberg** [ContainX/openstack4j#1299]
Why when trying to get the projects i am getting the following exception:
com.fasterxml.jackson.databind.JsonMappingException: Can not deserialize instance of java.lang.String out of START_OBJECT token
i am using the latest 3.2.0 jar

**Comment by LmV4corbeau** [comment]
The same Issue can be seen in the 3.2.1-Snapshot

**Comment by LmV4corbeau** [comment]
Resolves in new openstack4j repo

**Comment by cfsnyder** [comment]
I also ran into this issue after upgrading to Train. @LmV4corbeau what did you mean when you said it resolves in the "new" openstack4j repo?

**Comment by gilbirenberg** [comment]
Its look like there is 3.5 version where this is fix, look for this version, OpenStack4j change the site
They have also a site on https://openstack4j.github.io/
Here you will fine the new version
Hope this will help
Gil

נשלח מ-Workspace ONE Boxer

בתאריך 17 ביוני 2020 21:25, Cory Snyder <notifications@github.com> כתב:

I also ran into this issue after upgrading to Train. @LmV4corbeau<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_LmV4corbeau&d=DwMCaQ&c=UrUhmHsiTVT5qkaA4d_oSzcamb9hmamiCDMzBAEwC7E&r=8q2pjG-VwvID0B7UjwglCy4bGLAUJ9KpECaTX1Hi9BA&m=iloWey6zhGJKQcHfO9_4MfGBlZoQsolUs5xvKlDsr6U&s=UdhPBG2paD_t7sc7-3psLOZxrsnxNbjlwXoEpkJRzIU&e=> what did you mean when you said it resolves in the "new" openstack4j repo?

-

**Comment by cfsnyder** [comment]
Thanks @gilbirenberg , I confirm that it is fixed on v3.6 in the new repo! I'm surprised that there is no mention of the new repo in the README of this project (unless I'm missing it).

**Comment by olivergondza** [comment]
Yeah, https://github.com/openstack4j/openstack4j/ is struggling to get the old maintainers to either get the project ownership fully transferred or advertise the fork. I am sorry about the experience...
